src/V3Ast.h

```diff
@@ -2344,6 +2354,9 @@ class AstNodeBlock VL_NOT_FINAL : public AstNode {
     AstNode* stmtsp() const { return op1p(); }  // op1 = List of statements
     void addStmtsp(AstNode* nodep) { addNOp1p(nodep); }
     bool unnamed() const { return m_unnamed; }
+    virtual bool isFirstInOneOfMyListOfStatements(AstNode* n) const override {
+        return (n != nullptr && n == stmtsp());
+    }
```

Comment:
```suggestion
    virtual bool isFirstInOneOfMyListOfStatements(AstNode* nodep) const override {
        return (nodep && nodep == stmtsp());
    }
```

And rename method name, likewise elsewhere